src/client/pages/OfferNew/Checkout/SignDisclaimer.tsx

```diff
@@ -21,54 +22,75 @@ const Wrapper = styled('div')`
     margin: 0;
   }
 `
+
+export const getTermsLinks = (currentLocale: string) => {
+  const baseUrl = `https://www.hedvig.com/${currentLocale}`
+
+  const isPageInEnglish = currentLocale.includes('en')
+
+  const termslinkSE = isPageInEnglish
+    ? `${baseUrl}/terms`
+    : `${baseUrl}/villkor`
+
+  if (currentLocale.includes('se')) {
+    return termslinkSE
+  }
+
+  if (currentLocale.includes('no')) {
+    return `${baseUrl}/terms`
+  }
+
+  return ''
+}
```

Comment:
Honestly i'd prefer "less clever" than this. Feels shaky. Something like 

```ts
switch (currentLocale) {
  case 'se':
    return baseUrl + '/se/villkor'
  case 'no':
    return baseUrl + '/no/terms'
  // ...
}
```

feels easier to understand what's going on or?

Reply:
Sure, you're probably right. This is highly temporary though, which I've tried to be super clear about. So I don't thinkg it's worth taking the time to change it.

Reply:
Do we have a plan on when to remove it? (genuine question) Otherwise I think its not temporary enough, if you get what i mean :D

Reply:
I think the plan is to remove it when we start signing people on Hedvig's books? But I'm not sure

Reply:
OK so I updated the function the way you wanted